packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -182,7 +182,7 @@ export default function lwc(pluginOptions: RollupLwcOptions = {}): Plugin {
 
     return {
         name: PLUGIN_NAME,
-
+        version: process.env.LWC_VERSION,
```

Comment:
```suggestion
        // The version from the package.json is inlined by the build script
        version: process.env.LWC_VERSION,
```

Reply:
I need to kick this habit of commenting on github instead of in the code 🤦 . Thanks!